dotnet/targets/Microsoft.MaciOS.Sdk.Xcode.targets

```diff
@@ -0,0 +1,205 @@
+<!--
+***********************************************************************************************
+Microsoft.MaciOS.Sdk.Xcode.targets
+
+This file contains MSBuild targets that support building and operating on Xcode projects.
+
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="Sharpie"                AssemblyFile="$(_XamarinTaskAssembly)"/>
+  <UsingTask TaskName="XcodeBuild"             AssemblyFile="$(_XamarinTaskAssembly)"/>
+
+  <PropertyGroup>
+    <_MaciOSXcodeProjectDefaultOutputPathRoot>$(MSBuildProjectDirectory)/$(IntermediateOutputPath)xcode/</_MaciOSXcodeProjectDefaultOutputPathRoot>
+    <_BuildMaciOSXcodeProjectsStamp>$(_MaciOSStampDirectory)_BuildMaciOSXcodeProjects.stamp</_BuildMaciOSXcodeProjectsStamp>
+    <_SharpieBindMaciOSXcodeProjectsStamp>$(_MaciOSStampDirectory)_SharpieBindMaciOSXcodeProjects.stamp</_SharpieBindMaciOSXcodeProjectsStamp>
+    <_XcArchiveExtraArgs>$(_XcArchiveExtraArgs) ENABLE_BITCODE=NO SKIP_INSTALL=NO SWIFT_INSTALL_OBJC_HEADER=YES BUILD_LIBRARY_FOR_DISTRIBUTION=YES</_XcArchiveExtraArgs>
+    <_XcArchiveExtraArgs>$(_XcArchiveExtraArgs) OTHER_LDFLAGS=&quot;-ObjC&quot; OTHER_SWIFT_FLAGS=&quot;-no-verify-emitted-module-interface&quot; OBJC_CFLAGS=&quot;-fno-objc-msgsend-selector-stubs -ObjC&quot;</_XcArchiveExtraArgs>
+  </PropertyGroup>
+
+  <ItemDefinitionGroup>
+    <MaciOSXcodeProject>
```

Comment:
I'm not sure about the `@(MaciOSXcodeProject)` item name, does anyone have other suggestions? We went with `@(AndroidGradleProject)` for the Android counterpart of these changes.

Reply:
I think just `@(XcodeProject)` sounds good. It doesn't look like there's much other code it will clash with: https://github.com/search?q=path%3A*.targets+XcodeProject&type=code&ref=advsearch

Reply:
Renamed to @(XcodeProject)